datasets/wiki_summary/wiki_summary.py

```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Wiki Summary."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+The dataset extracted from Persian Wikipedia into the form of articles and highlights and cleaned the dataset into pairs of articles and highlights and reduced the articles' length (only version 1.0.0) and highlights' length to a maximum of 512 and 128, respectively, suitable for parsBERT.
+"""
+
+_CITATION = """\
+@misc{Bert2BertWikiSummaryPersian,
+  author = {Mehrdad Farahani},
+  title = {Summarization using Bert2Bert model on WikiSummary dataset},
+  year = {2020},
+  publisher = {GitHub},
+  journal = {GitHub repository},
+  howpublished = {https://github.com/m3hrdadfi/wiki-summary},
+}
+"""
+
+_TRAIN_FILENAME = "train.csv"
+_TEST_FILENAME = "test.csv"
+_DEV_FILENAME = "dev.csv"
+
+_ID = "id"
+_LINK = "link"
+_TITLE = "title"
+_ARTICLE = "article"
+_HIGHLIGHTS = "highlights"
+
+_HOMEPAGE = "https://github.com/m3hrdadfi/wiki-summary"
+
+
+class WikiSummary(datasets.GeneratorBasedBuilder):
+    """Wiki Summary"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+    You need to go to https://github.com/m3hrdadfi/wiki-summary,
```

Comment:
Did you try using the download manager with those urls ?
- train: https://drive.google.com/u/0/uc?id=1-CaP3xHgZxOGjQ3pXC5tr9YnIajmel-t&export=download
- dev: https://drive.google.com/u/0/uc?id=1-2g2gkDeNaN-vth-8Mgit_ovmSkVh91u&export=download
- test: https://drive.google.com/u/0/uc?id=1-9G4yYP6YO8oMA-o4cTe9NJpEyr7x5jg&export=download

It would be convenient to not have to download them manually

Reply:
yes I tried them but it didn't work,

Reply:
(Note that the urls I provided are different from the ones on the dataset repository, which are not urls for direct download and therefore don't work with the download manager)

Reply:
yes I tried the urls and now they are working, I will modify the code soon